ark/attest/cache/snapshots.ts

```diff
@@ -151,12 +151,18 @@ const snapshotArgsToQueuedUpdate = ({
 	baselinePath
 }: SnapshotArgs): QueuedUpdate => {
 	const snapCall = findCallExpressionAncestor(position, snapFunctionName)
-	const newArgText =
+	let newArgText =
 		typeof serializedValue === "string" && serializedValue.includes("\n") ?
 			"`" +
 			serializedValue.replaceAll("`", "\\`").replaceAll("${", "\\${") +
 			"`"
 		:	JSON.stringify(serializedValue)
+
+	if (newArgText.includes("$ark.bigint-"))
```

Comment:
We would want `startsWith` rather than includes here. I'd probably put the prefixes in constant variables at the module level, check if the string starts with them.

If it does, do `s = s.slice(prefix.length) + "n"`

Reply:
fixed! - it was a lil different, but I followed the spirit of your message here.